src/Console/Command/DescribeCommand.php

```diff
@@ -204,24 +228,119 @@ private function arrayToText(array $data)
     private function prepareDiff($isDecoratedOutput, $diff)
     {
-        $template = "<comment>   ---------- begin diff ----------</comment>\n%s\n<comment>   ----------- end diff -----------</comment>";
-
-        $diff = implode(
-            "\n",
-            array_map(
-                function ($string) {
-                    $string = preg_replace('/^(\+.*)/', '<fg=green>\1</>', $string);
-                    $string = preg_replace('/^(\-.*)/', '<fg=red>\1</>', $string);
-                    $string = preg_replace('/^(@.*)/', '<fg=cyan>\1</>', $string);
-
-                    return '   '.$string;
-                },
-                preg_split("#\n\r|\n#", OutputFormatter::escape(rtrim($diff)))
-            )
-        );
+        if ($isDecoratedOutput) {
+            $template = "<comment>   ---------- begin diff ----------</comment>\n%s\n<comment>   ----------- end diff -----------</comment>";
+            $diff = implode(
+                PHP_EOL,
+                array_map(
+                    function ($string) {
+                        $string = preg_replace('/^(\+.*)/', '<fg=green>\1</>', $string);
+                        $string = preg_replace('/^(\-.*)/', '<fg=red>\1</>', $string);
+                        $string = preg_replace('/^(@.*)/', '<fg=cyan>\1</>', $string);
+
+                        return '   '.$string;
+                    },
+                    preg_split("#\n\r|\n#", OutputFormatter::escape(rtrim($diff)))
+                )
+            );
+        } else {
+            $template = '      ---------- begin diff ----------%s      ----------- end diff -----------';
+        }
 
         return sprintf($template, $diff);
     }
+
+    /**
+     * @return array<string, FixerInterface>
+     */
+    private function getFixers()
+    {
+        if (null !== $this->fixers) {
+            return $this->fixers;
+        }
+
+        $fixerFactory = new FixerFactory();
+        $fixers = array();
+
+        foreach ($fixerFactory->registerBuiltInFixers()->getFixers() as $fixer) {
+            $fixers[$fixer->getName()] = $fixer;
+        }
+
+        $this->fixers = $fixers;
+        ksort($this->fixers);
+
+        return $this->fixers;
+    }
+
+    /**
+     * @return string[]
+     */
+    private function getSetNames()
+    {
+        if (null !== $this->setNames) {
+            return $this->setNames;
+        }
+
+        $set = new RuleSet();
+        $this->setNames = $set->getSetDefinitionNames();
+        sort($this->setNames);
+
+        return $this->setNames;
+    }
+
+    /**
+     * @param string $type 'rule'|'set'
+     * @param string $name
+     *
+     * @return string|null
+     */
+    private function getAlternative($type, $name)
+    {
+        $other = null;
+        $alternatives = 'set' === $type ? $this->getSetNames() : $this->getFixers();
+
+        foreach ($alternatives as $alternative) {
+            $distance = levenshtein($name, $alternative);
+            if ($distance < 2) {
```

Comment:
I think SF default is 3 for commands, its all about feeling I guess between false positive and missed opportunity.